build.gradle

```diff
@@ -1222,6 +1225,42 @@ project(':metadata') {
   }
 }
 
+project(':group-coordinator') {
+  archivesBaseName = "kafka-group-coordinator"
+
+  dependencies {
+    implementation project(':server-common')
+    implementation project(':clients')
+    implementation libs.slf4jApi
+
+    testImplementation project(':clients')
```

Comment:
Isn't this redundant given that we already have an implementation dependency on clients? Or does gradle complain if you remove it?

Reply:
You're right. Gradle does not complain without them. I removed them.